www/index.html

```diff
@@ -55,7 +56,7 @@
         <section id="search-article" role="region">
             <header id="top">
                 <nav class="navbar navbar-expand-md bg-light" role="navigation">
-                    <a class="navbar-brand">Kiwix 3.3-WIP</a>
+                    <a id="appVersion" class="navbar-brand">Kiwix</a>
```

Comment:
Now setting this dynamically from the appVersion defined in init.js. We will need to change script that updates this programmatically when preparing a new release.

Reply:
Done.